bin/build-hdfs

```diff
@@ -91,3 +93,15 @@ cd ..
 echo "Compressing to $DIST.tgz"
 rm -f $DIST.tgz
 tar czf $DIST.tgz $DIST
+
+
+# Build docker container
+if [ "$1" == "docker" ] && [ -f $DIST.tgz ]; then
+    if [ ! -z "$2" ]; then
```

Comment:
[ ! -z "foo" ] is equivalent to [ "foo" ], or you can use -n if you want to explicitly test for a non-empty string.